tools/data-handler/src/interfaces/project-interfaces.ts

```diff
@@ -215,23 +231,26 @@ export interface fetchCardDetails {
   parent?: boolean;
 }
 
-export type metadataContent =
+//
+export type MetadataContent =
   | number
   | boolean
   | string
   | string[]
-  | link[]
+  | Link[]
   | null
   | undefined;
 
-export type csvRowRaw = {
+//
+export type CSVRowRaw = {
   [key: string]: string;
 };
 
-export interface importCsvRow {
-  title: string;
-  template: string;
-  description?: string;
-  labels?: string[];
-  [key: string]: string | string[] | undefined; // Custom fields
-}
+// todo: this is apparently not used?
```

Comment:
If there is some reason why this is apparently unused, I can keep it. Otherwise, before merging this PR, I will delete it.

Reply:
I think it can be deleted